### PR TITLE
Create Timely-Greeting.js

### PR DESCRIPTION
**Pull request by Gordon-BP**
New custom action that automatically greets the user with "Good morning/afternoon/evening" based on the time of day.

### CODE REVIEW

File: custom solutions/Timely Greeting/Timely-Greeting.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        .zone(user.timezone.toString())
+        .format('HH')
+    )
+    bp.logger.info(`current time is ${currentTime}`)
+    bp.logger.info(`Morning ends at ${schedule.morningEnd} and afternoon ends at ${schedule.afternoonEnd}`)
+
+    if (currentTime >= schedule.morningStart && currentTime <= schedule.morningEnd) {
+      message = messages[0]
+    }
+    if (currentTime > schedule.morningEnd && currentTime <= schedule.afternoonEnd) {
```
Comment (davidvitora) [inline custom solutions/Timely Greeting/Timely-Greeting.js:62]:
```suggestion
    else if (currentTime > schedule.morningEnd && currentTime <= schedule.afternoonEnd) {
```

File: custom solutions/Timely Greeting/Timely-Greeting.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      message = messages[1]
+    }
+    if (currentTime > schedule.afternoonEnd || currentTime <= schedule.morningStart) {
```
Comment (davidvitora) [inline custom solutions/Timely Greeting/Timely-Greeting.js:65]:
```suggestion
    else if (currentTime > schedule.afternoonEnd || currentTime <= schedule.morningStart) {
```

**Review by davidvitora — APPROVED** [review]
few suggestions, good job on this

**Comment by Gordon-BP** [comment]
Thanks for the review!

**Pull request merged**